samples/uart_echo.cpp

```diff
@@ -20,10 +20,10 @@ g++ samples/uart_echo.cpp src/UartSerial.cpp -I include -pthread -std=c++11 -Wal
 
 */
 
-#include <Stream.h>
-#include <UartSerial.h>
 #include <unistd.h>
 #include <iostream>
+#include "Stream.h"
```

Comment:
The sample is not part of the library, it is a consumer of the library. So, "Stream.h" and "UartSerial.h" are actually external to your sample, so these were already "correct".